README.rst

```diff
@@ -2,33 +2,33 @@
 ResolveLib
 ==========
 
-ResolveLib at the highest level provides a `Resolver` class that includes
+ResolveLib at the highest level provides a ``Resolver`` class that includes
 dependency resolution logic. You give it some things, and a little information
 on how it should interact with them, and it will spit out a resolution result.
 
 
 Intended Usage
 ==============
 
-::
+.. code-block:: python
```

Comment:
The readme intentionally only uses built-in RestructureText syntaxes since it may have compatibility issues when the project is uploaded. `.. code-block::` is a Sphinx extension.

Reply:
GitHub and PyPI take `code-block` just fine, however I'm not entirely sure about other rendering engines.  A plus point for `code-block` is that some editors may allow syntax highlighting as well, e.g.
![2020-05-31T11:05:57](https://user-images.githubusercontent.com/13689192/83344275-b80e7600-a32e-11ea-90cb-171b10c0c2c3.png)

That being said, I'll still revert the change since the highlighting we have here is minimal.